src/QA/QA.Web/Views/Review/Review.cshtml

```diff
@@ -6,31 +6,49 @@
 }
 <div class="review-container">
     <form asp-route="@postRoute">
+
         <div class="grid-row">
-            <div asp-validation-summary="All" class="error-summary" style="display:@(ViewData.ModelState.IsValid ? "none" : "block")"></div>
+            <div class="column-full">
```

Comment:
can you put an `asp-condition="@ViewData.ModelState.IsValid" attribute on this div so we don't need the `style` attribute

Reply:
Done